ledger/sandbox-classic/src/main/scala/platform/sandbox/stores/ledger/ScenarioLoader.scala

```diff
@@ -160,10 +159,11 @@ private[sandbox] object ScenarioLoader {
         val pkg = packages
           .getLfPackageSync(packageId)
           .getOrElse(sys.error(s"Listed package $packageId not found"))
-        pkg.lookupDefinition(scenarioQualName) match {
-          case Right(x) =>
-            List((Ref.Identifier(packageId, scenarioQualName) -> x))
-          case Left(_) => List()
+        pkg.modules
```

Comment:
because we need the value attached to the definition, Interface returns `GenDValue[_]` while here we need  `GenDValue[Expr]`. It is actually the only place where we do this kind of lookup.